js/scripts.js

```diff
@@ -28,42 +27,41 @@
 	const flipACoin = function() {
 		let randomFlip = Math.random()
 		if (randomFlip < .5) {
-			coin = ["heads", "tails"];
+			return "heads";
 		} else {
-			coin = ["tails", "heads"];
+			return "tails"
```

Comment:
We only need to return a single string that represents whether the result was heads or tails. No need for an Array.